my-app/src/components/organisms/movie-manipulation/movie_manipulation.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import { ButtonInfo } from '../../atoms/button-info/button-info';
+import { Button } from '../../atoms/button/button';
+import { Modal } from '../modal/modal';
+import { AddMovieModal } from '../add-movie-modal/add-movie-modal';
+import { oneFilmData } from '../../../toolkit-store/index'
+
+interface DeleteMovieProps {
+    filmId: number;
+    className?: string;
+    filmsArray: oneFilmData[];
+}
+
+export const MovieManipulation = ({ className, filmId, filmsArray }: DeleteMovieProps) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isDeleted, setDeleted] = useState(false);
+    const [isUpdateButtonsShow, setIsUpdateButtonsShow] = useState(false);
+    const [isUpdateMovieModalOpen, setIsUpdateMovieModalOpen] = useState(false);
+    const [dataForUpdate, setDataForUpdate] = useState(null)
+
+
+    const handlerShowDeleteModal = () => {
+        setIsOpen(true);
+        handlerShowUpdateButtons();
+    }
+    const handlerDelete = async () => {
+        await fetch(`http://localhost:4000/movies/${filmId}`, { method: 'DELETE' });
```

Comment:
You didn't refetch the updated movies in the state, so the best way would be to have a separate redux action for it to delete and remove the movie from state

Reply:
The same is for add movie, just redux action that will add the movie to state

Reply:
Done